large_media

```diff
@@ -1 +1 @@
-Subproject commit bd07d54c44b9ee8ff381707e1117e365d8079096
+Subproject commit 65977abfe982c9b00a93eaa52e47c169d9c28085
```

Comment:
**Caution!** This contains a submodule update

Reply:
@GiudGiud not sure what the protocol is for a submodule update when it picks up on other commits that were added in between the current large_media commit that moose uses and the most recent large_media commit that has the updated images for this PR. Should I be modifying the commit history so that the other updates to `large_media/tensor_mechanics` are removed?

Reply:
no it's fine to bring in other people's images. thanks for checking